test/tsk/vstools/test_mmls.cpp

```diff
@@ -0,0 +1,114 @@
+#include <fstream>
```

Comment:
Tests for `tools/vstools/mmls.cpp` should be in `test/tools/vstools/test_mmls.cpp`.

Reply:
Noted. Thanks.